test/exchange_wrapper_test.ts

```diff
@@ -323,6 +468,26 @@ describe('ExchangeWrapper', () => {
                     expect(cancelledAmount).to.be.bignumber.equal(cancelAmount);
                 });
             });
+            describe('order transaction options', () => {
+                it('should validate when orderTransactionOptions are not present', async () => {
+                    return expect(zeroEx.exchange.cancelOrderAsync(signedOrder, new BigNumber(0)))
```

Comment:
Nit: Let's not in-line instantiating new objects in method params. Instead declare outside with a descriptive variable name of what it represents.

Our convention is also not to pass numbers/strings/booleans directly at params but to always assign them to variables with readable names. Otherwise the reader needs to look at the method definition to figure out what the value represents (saves future devs time).